### PR TITLE
chore: In integ_test don't run unit test

### PR DESCRIPTION
**Pull request by tmdzk**
Signed-off-by: Timothée Dzik <timdzik@fb.com>

chore: In integ_test don't run unit test

## Summary

We're already running them on the agw workflow, no need to run them while we do the integ_test
https://github.com/magma/magma/blob/master/.github/workflows/agw-workflow.yml#L108

### CODE REVIEW

**Comment by tmdzk** [comment]
Tests are passing: https://github.com/magma/magma/runs/4438216314?check_suite_focus=true even with the flag

**Review by quentinDERORY — CHANGES_REQUESTED** [review]
Looks like you are skipping them in both places?

**Review by quentinDERORY — APPROVED** [review]

File: lte/gateway/Makefile [hunk 1]
```diff
@@ -202,7 +202,7 @@ scan_oai: ## Scan OAI
 test_python: stop ## Run all Python-specific tests
 	make -C $(MAGMA_ROOT)/lte/gateway/python test_all
 
-test_sudo_python: stop ## Run Python tests that require sudo (datapath, etc.)
+test_sudo_python: stop ## Run Python tests that require sudo (datapath, etc.) hello
```
Comment (ardzoht) [inline lte/gateway/Makefile]: ??

**Review by ardzoht — APPROVED** [review]
lgtm

File: .github/workflows/agw-workflow.yml [hunk 2]
```diff
@@ -106,6 +106,9 @@ jobs:
       - name: Execute test_all
         run: |
             make -C ${{ env.MAGMA_ROOT }}/lte/gateway/python test_all
+      - name: Execute test_all with skip flag
+        run: |
+            make -C ${{ env.MAGMA_ROOT }}/lte/gateway/python test_all SKIP_NON_SUDO_TESTS=1
```
Comment (themarwhal) [inline .github/workflows/agw-workflow.yml]: Did you try removing the env var above? `SKIP_SUDO_TESTS: 1`

File: lte/gateway/fabfile.py [hunk 3]
```diff
@@ -256,7 +256,6 @@ def integ_test(
 
     execute(_dist_upgrade)
     execute(_build_magma)
-    execute(_run_sudo_python_unit_tests)
```
Comment (themarwhal) [inline lte/gateway/fabfile.py:259]: If we're committing this let's just remove the function from the file too

**Pull request closed without merging**